chain/default_syncer.go

```diff
@@ -110,7 +110,7 @@ func (syncer *DefaultSyncer) getBlksMaybeFromNet(ctx context.Context, blkCids []
 // that interacts with the network. It does NOT add tipsets to the chainStore..
 func (syncer *DefaultSyncer) collectChain(ctx context.Context, tipsetCids types.SortedCidSet) ([]types.TipSet, error) {
 	var chain []types.TipSet
-	defer logSyncer.Info("chain synced")
+	defer logSyncer.Info("chain fetch from network complete")
```

Comment:
Could be good to include `tipsetCids` in this log so it is easy to identify the fetched chain as the same one referred to earlier.

Reply:
(original tipsetCids of the head that is, not the value this variable updates to each loop